web/src/compose_banner.ts

```diff
@@ -65,6 +67,10 @@ export function clear_warnings(): void {
     $(`#compose_banners .${CSS.escape(WARNING)}`).remove();
 }
 
+export function clear_unmute_topic_notifications(): void {
+    $(`#compose_banners .${CLASSNAMES.unmute_topic_notification.replaceAll(" ", ".")}`).remove();
+}
```

Comment:
Why does this need the `replaceall`? Also, this shouldn't skip the `CSS.escape` pattern we're using elsewhere in this file.

Reply:
This variable is a string of two classes separated with a space, while all others are a single class (so no spaces). The variable works as is for use in the template but as a CSS selector we need to replace the space with a period.

The reason I skipped escaping this was that it would add an unneeded backslash before the space.

Reply:
OK, this makes sense.